src/frontend/src/btc/services/btc-listener.services.ts

```diff
@@ -56,7 +56,7 @@ export const syncWalletError = ({
 	// Currently, the certified error can only happen while fetching balance, but we still want to reset transactions to avoid displaying incorrect data
 	btcTransactionsStore.reset(tokenId);
 
-	if (!hideToast) {
+	if (hideToast) {
```

Comment:
Since we have discussed quite a bit about it and the fact that I missed it in the review (😅), can you provide a test for it (next week)?

Reply:
Absolutely, that was also my thought after I noticed it a few mins ago 🙃